pkg/queue/stats.go

```diff
@@ -63,11 +67,16 @@ func NewStats(podName string, channels Channels, startedAt time.Time) *Stats {
 	}
 
 	go func() {
-		var requestCount int32
-		var concurrency int32
+		var (
+			requestCount int32
+			proxiedCount int32
+			concurrency  int32
+			proxiedCon   int32
```

Comment:
Let's name this `proxiedConcurrency` to be crystal clear in the code itself.

Reply:
done.